bin/i18n/sync-out.rb

```diff
@@ -239,18 +255,22 @@ def distribute_course_content(locale)
   end
 
   locale_strings.each do |type, translations|
-    type_data = Hash.new
-    type_data[locale] = Hash.new
-    type_data[locale]["data"] = Hash.new
-    type_data[locale]["data"][type] = translations.sort.to_h
-
     # We'd like in the long term for all of our generated course content locale
     # files to be in JSON rather than YAML. As a first step on that journey,
     # here we serialize all of the locale types except DSLs to JSON. The DSL
     # locale file is unfortunately touched by a few other processes, so
     # converting that one over will have to be done as part of a larger effort.
     extension = type == "dsls" ? "yml" : "json"
-    sanitize_data_and_write(type_data, "dashboard/config/locales/#{type}.#{locale}.#{extension}")
+    type_file = "dashboard/config/locales/#{type}.#{locale}.#{extension}"
+
+    existing_data = YAML.load_file(type_file).dig(locale, "data", type) || {}
```

Comment:
Aren't these json files now?

Reply:
d'oh! That's what I get for testing this before the other changes out there got merged